mkcli.py

```diff
@@ -182,7 +196,7 @@ def run(args):
 
         testsExecuted = gatherFeedbackData(browserName)
         url = muuktestRoute+'feedback/'
-        values = {'tests': testsExecuted, 'userId': userId, 'browser': browserName}
+        values = {'tests': testsExecuted, 'userId': userId, 'browser': browserName,'executionNumber': int(executionNumber)}
```

Comment:
what happen if executionNumber is None because it was not provided?

Reply:
it will send NaN to the BE, execution wont be found and the feedback won't be saved